README.md

```diff
@@ -66,6 +67,8 @@ pnpm i
 
 ### Local Playground
 
+Go to the `.env.example` file and make a copy of it named `.env` file. Copy the Access Keys from your Rapyd Sandbox account into `.env` file.
```

Comment:
What are your thoughts on this approach of specifying the environment variables? It might also be beneficial to explicitly mention the environment variables present in the example file.

```suggestion
### Environment Variables
To ensure the proper functioning of the local playground, it is necessary to configure the environment variables appropriately. Additionally, you must duplicate the example environment file into your local environment file. Please note that this file is not tracked in the version control system and should not be included in any commits.
Navigate to the project's root directory and enter the following command:

```sh cp ./docker/dev/.env.example ./docker/dev/.env```

Using your preferred text editor, open the `./docker/dev/.env` file and configure the necessary environment variables.
```